src/tetris/player/localPlayer.ts

```diff
@@ -17,51 +17,57 @@ export default class LocalPlayer extends Player {
 		// update bias events
 		this._biasEventReceiver.update(time, delta);
 		// apply bias
-		if (this._biasEventReceiver.has(BiasEvenType.DisableInput)) {
+		if (this._biasEventReceiver.has(BiasEventType.DisableInput)) {
 			return;
 		}
 		let moveOperation: () => void;
 
+
 		// handle keyboard inputs
-		if(this._cursor.left.isDown) {
-			moveOperation = this.moveLeft;
-		} else if (this._cursor.right.isDown) {
-			moveOperation = this.moveRight;
-		} else if (this._cursor.down.isDown) {
-			moveOperation = this.moveDown;
-		} else if (this._cursor.space.isDown) {
-			moveOperation = this.drop;
-		} else if (this._cursor.up.isDown) {
-			moveOperation = this.rotate;
+		if (Phaser.Input.Keyboard.JustDown(this._cursorKeys.left)) {
```

Comment:
Why `JustDown()`? I had a look into the phase 3 documentation / tutorial and they suggested to use isDown.